content/millennium/dstu2/diagnostic/diagnostic-report.md

```diff
@@ -26,7 +26,10 @@ The following fields are returned if valued:
 * [Issued date/time](http://hl7.org/fhir/DSTU2/diagnosticreport-definitions.html#DiagnosticReport.issued){:target="_blank"}
 * [Performer (practitioner)](http://hl7.org/fhir/DSTU2/diagnosticreport-definitions.html#DiagnosticReport.performer){:target="_blank"}
 * [Request (order)](http://hl7.org/fhir/DSTU2/diagnosticreport-definitions.html#DiagnosticReport.request){:target="_blank"}
-* [ContentType and URL (fully qualified link to the document](http://hl7.org/fhir/DSTU2/diagnosticreport-definitions.html#DiagnosticReport.presentedForm){:target="_blank"}
+* [Presented Form](http://hl7.org/fhir/DSTU2/diagnosticreport-definitions.html#DiagnosticReport.presentedForm){:target="_blank"}
+  * [ContentType](http://hl7.org/fhir/DSTU2/datatypes-definitions.html#Attachment.contentType){:target="_blank"}
```

Comment:
the other one says Attachment ContentType, probably we should just change both to 'Content type'

Reply:
[Updated](https://github.com/cerner/fhir.cerner.com/pull/338/commits/a74a052b80d6ebd14b13ee8fb007ba0a561426c2)